### PR TITLE
Part1 of Project2.

### PR DESCRIPTION
**Pull request by WingTangWong**
* Added primary data types
* Added functions
* Added calls to functions with variables being ignored.
* Tested the code run and it is clean.

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -16,29 +16,41 @@ video: Chapter 2 - Part 3
...
+
+
+
+   * Integer 
+   * Character
+   * Boolean
+   * Floating Point
+   * Double Floating Point
+   * Void 
+   * Wide Character
```
Comment (matkatmusic) [inline main.cpp]: - [x] please write down the C++ type name, not the english name

File: main.cpp [hunk 2]
```diff
@@ -65,11 +79,56 @@ void variableDeclarations()
...
+    ignoreUnused(letterOfFailGrade);
+    ignoreUnused(isItemBurning);
+    ignoreUnused(isItemWet);
+    ignoreUnused(isItemFull);
+    ignoreUnused(realPiValue);
+    ignoreUnused(realRepeatValue);
+    ignoreUnused(realPercentage);
+    ignoreUnused(longerPiValue);
+    ignoreUnused(longerRepeatValue);
+    ignoreUnused(longerPercentage);
```
Comment (matkatmusic) [inline main.cpp]: - [x] pass them all to a single call of `ignoreUnused()`, like in the `rentACar()` implementation.

**Review by matkatmusic — CHANGES_REQUESTED** [review]
Every function has a default value for each function parameter.  Can you remove a few?  You could potentially call every function in `main()` without passing any parameters.

File: main.cpp (hunk 1 above)
Comment (WingTangWong) [reply main.cpp]: Done. Changed to c++ primitive names.

File: main.cpp (hunk 2 above)
Comment (WingTangWong) [reply main.cpp]: Done. Collapsed into single call.

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**